crates/bevy_window/src/event.rs

```diff
@@ -58,10 +58,21 @@ pub struct WindowCloseRequested {
 pub struct WindowClosed {
     pub id: WindowId,
 }
-/// An event that is sent whenenver the user's cursor moves.
+/// An event reporting that the mouse cursor has moved on a window.
+///
+/// The event is sent only if the cursor is over one of the application's windows.
+/// It is the translated version of [`WindowEvent::CursorMoved`] from the `winit` crate.
+///
+/// Not to be confused with the [`MouseMotion`] event from `bevy_input`.
+///
+/// [`WindowEvent::CursorMoved`]: https://docs.rs/winit/latest/winit/event/enum.WindowEvent.html#variant.CursorMoved
+/// [`MouseMotion`]: bevy_input::mouse::MouseMotion
 #[derive(Debug, Clone)]
 pub struct CursorMoved {
+    /// The identifier of the window the cursor has moved on.
     pub id: WindowId,
+
+    /// The position of the cursor, in window coordinates.
```

Comment:
This would be absolute position (in the window) and not delta from last event, yes?

Reply:
Yes, it's correct